routes/userRouter.js

```diff
@@ -0,0 +1,42 @@
+const { Router }=require('express');
+const { UserModel } = require('../MongoDBModels/User');
+
+const userRouter = new Router()
+
+userRouter.post('/user', async (req, res) => {
+
+    try {
+        const userData = req.body
+
+        const user = await UserModel.create(userData)
+
+        res.status(201).json({
+            user
+        })
+    } catch (e) {
+        res.status(500).json({
+            error: e.message
+        })
+    }
+})
+
+userRouter.get('/user', async (req, res) => {
+
+    try {
+
+        const id = req.query.id;
```

Comment:
should be validated because for  this request ```curl --location 'http://localhost:3000/user?id=4' \
--header 'Content-Type: application/json'``` 
  
I got next error
`{"error": "Cast to ObjectId failed for value \"4\" (type string) at path \"_id\" for model \"User\""}`

Error handled, but due to catch, should be handled before request reach the handler but you can leave it for now its is ok 👍